plugins/modules/keycloak_client.py

```diff
@@ -805,9 +805,6 @@ def normalise_cr(clientrep, remove_ids=False):
     # Avoid the dict passed in to be modified
     clientrep = clientrep.copy()
 
-    if 'attributes' in clientrep:
-        clientrep['attributes'] = list(sorted(clientrep['attributes']))
```

Comment:
Is it possible that this used to be a list, and was changed to a dict at some point?

Reply:
It kinda looks like it. But i checked Keycloak 5, 9, 13, 18, 25 and its always a dict. Same for the clientscopes.

Reply:
This was added in #3610 by @pmdumuid. @pmdumuid do you remember why you chose this? The tests added in the PR use Keycloak 12.0.4, maybe that version uses it for whatever reason (even if 9 and 13 do not)?